src/main/java/com/artipie/rpm/benchmarks/RpmBench.java

```diff
@@ -71,13 +79,52 @@ public void setup() {
         if (RpmBench.BENCH_DIR == null) {
             throw new IllegalStateException("BENCH_DIR environment variable must be set");
         }
-        this.storage = new FileStorage(Paths.get(RpmBench.BENCH_DIR));
+        this.storage = new InMemoryStorage();
```

Comment:
@g4s8 as we want to measure memory consumption later, I'm not sure that using `InMemoryStorage` is a good idea

Reply:
@olenagerasimova in any case we need to measure memory consumption by some other tool - not JMH, so we will need to write yet another test for that.